vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/BiodataVariantToVariantContextConverter.java

```diff
@@ -79,6 +86,43 @@ public VariantContext transform(Variant variant) {
         return variantContext;
     }
 
+    private String getAnnotationAttributes(Variant variant) {
+        List<ConsequenceType> consequenceTypes = variant.getAnnotation().getConsequenceTypes();
+        String allele = variant.getAnnotation().getAlternativeAllele();
+        String csq = "";
+        if (consequenceTypes != null) {
+            List<String> consequences = new ArrayList<>();
+
+            for (ConsequenceType consequenceType : consequenceTypes) {
+
+                List<ConsequenceType.ConsequenceTypeEntry> soTermList = consequenceType.getSoTerms();
+                List<String> soNameList = soTermList.stream().map(ConsequenceType.ConsequenceTypeEntry::getSoName)
+                        .collect(Collectors.toList());
+                String soNames = String.join("&", soNameList);
+                String symbol = consequenceType.getGeneName();
+                String gene = consequenceType.getEnsemblGeneId();
+                String feature = consequenceType.getEnsemblTranscriptId();
+                String bioType = consequenceType.getBiotype();
```

Comment:
I like how this method looks now, good work! However, I'm worried because it seems that some of the consequence types could be null (there is a ConsequenceType constructor that initializes cds to null, for example) , so I think that the resulting CSQ strings would be something like A|regulatory|null|anotherfield ... instead of A|regulatory||anotherfield. 

Can you check what is happening initializing some of the fields as null in the test?